### PR TITLE
mockup axit task 2 - JS

### PR DESCRIPTION
**Pull request by maycatsby**
https://maycatsby.github.io/ui-internship-lwo2/HTML/mockup-axit/

### CODE REVIEW

File: HTML/mockup-axit/js/content.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const paragraph = [
```
Comment (tkachukivan) [inline HTML/mockup-axit/js/content.js]: we have objects in js so why not use one structure like tab = { title: 'Title', description: 'sometext'}

File: HTML/mockup-axit/js/task2.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import {elementsLoaded} from './elements.js';
+import * as content from './content.js';
+
+export const addingTabsEffects = () => {
+  elementsLoaded.then((data) => {
+    const elements = data;
```
Comment (tkachukivan) [inline HTML/mockup-axit/js/task2.js]: destructuring?

File: HTML/mockup-axit/js/task2.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    const changeTabColor = (e) => {
+      elements.tabs.forEach((el) => {
+        el.classList.remove('tab--selected');
+      });
+      e.target.classList.add('tab--selected');
+    };
+
+    const addingClickListener = () => {
+      elements.tabs.forEach((el, index) => {
```
Comment (tkachukivan) [inline HTML/mockup-axit/js/task2.js]: event delegation could be great technic to use here

File: HTML/mockup-axit/js/task2.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        el.addEventListener('click', (e) => {
+          changeTabColor(e);
```
Comment (tkachukivan) [inline HTML/mockup-axit/js/task2.js]: probably changeActiveTab would be better

**Review by tkachukivan — CHANGES_REQUESTED** [review]
what about opacity animation for content :(

File: HTML/mockup-axit/js/task2.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import {elementsLoaded} from './elements.js';
+import * as content from './content.js';
+
+export const addingTabsEffects = () => {
```
Comment (tsvetkovv) [inline HTML/mockup-axit/js/task2.js]: `addTabsEffects`

**Review by tsvetkovv — CHANGES_REQUESTED** [review]
Storing static content in JS is not good idea. tab content should be placed in html, it also lets you to animate smoothly with opacity.
if you have objections/problems, ping me to discuss.

**Review by tkachukivan — APPROVED** [review]

**Review by tsvetkovv — APPROVED** [review]
good

**Pull request merged**